internal/bin/gethnet

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname "$0")"
+./print_account
+./topoff_gethnet
+
+if [ "$1" = "console" ]; then
+  CMD="geth console"
+else
+  CMD="geth"
+fi
```

Comment:
I believe devnet would blow away past state and start a fresh blockchain. On the one hand I liked the simplicity of getting a fresh chain... on the other hand you would have to recreate contracts and jobs on every restart.

A nice compromise might be to add a command to easily wipe away the previous blockchain if you want to start fresh.